src/components/Todo/TodoList.tsx

```diff
@@ -0,0 +1,52 @@
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  todos: Todo[],
+};
+
+const TodoList: React.FC<Props> = ({ todos }) => {
+  return (
+    <section className="todoapp__main" data-cy="TodoList">
+      {
+        todos.map((todo: Todo) => {
+          return (
+            <div
+              data-cy="Todo"
+              className={`todo ${todo.completed && 'completed'} `}
+              key={todo.id}
+            >
+              <label className="todo__status-label">
+                <input
+                  data-cy="TodoStatus"
+                  type="checkbox"
+                  className="todo__status"
+                  defaultChecked
+                />
+              </label>
+              <span
+                data-cy="TodoTitle"
+                className="todo__title"
+              >
+                {todo.title}
+              </span>
+              <button
+                type="button"
+                className="todo__remove"
+                data-cy="TodoDeleteButton"
+              >
+                ×
+              </button>
+
+              <div data-cy="TodoLoader" className="modal overlay">
+                <div className="modal-background has-background-white-ter" />
+                <div className="loader" />
+              </div>
+            </div>
```

Comment:
Would be great to move it to a separate component - `TodoListItem` for example. It's up to you